### ISSUE TITLE
3DS: Favorites not able to be started

### ISSUE THREAD

**Issue by Clydefrosch22** [libretro/RetroArch#5338]
The newly added option to set a game as a favorite works insofar, as the game is added to the list of favorites, however, after selecting the game, retroarch seems to expect me to select a core, however, it's not displaying any cores as it would when selecting a game through the normal Load Content menu

**Comment by RobLoach** [comment]
Missing the issue template. Mind adding the info below?

```
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

[Description of the bug]

### Expected behavior

[What you expected to happen]

### Actual behavior

[What is actually happening]

### Steps to reproduce the bug

1. [First step]
2. [Second step]
3. [and so on...]

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: [version/commit]

### Environment information

- OS: [The operating system you're running]
- Compiler: [In case you are running local builds]
```

**Comment by Clydefrosch22** [comment]
sorry, i felt it was unnecessary for this type of issue.

# Description
You can't start a game from the Favorites Option in the 3DS version of retroarch.

### Expected behavior
I expected to be able to start games quickly through the new favorite option

### Actual behavior
But when I try to go to the favorite option and select a game I added, then hit 'Run' in the next screen, where I should apparently chose a core to start the game with, there is no core to select.

### Steps to reproduce the bug

1. start game normally through content > folders > romfile > core
2. select 'add to favorites'
3. unload game/close retroarch and restart it
4. go to content > favorites > game you just added > run > ???? 
5. no way to start the game here.


### Bisect Results

since the option was introduced in 1.6.6

### Version/Commit
You can find this information under Information/System Information

- RetroArch:  1.6.6 and 1.6.7

### Environment information

- OS: new 3ds on latest firmware with luma cfw

**Comment by danieljg** [comment]
@Clydefrosch22 are you using the cia or 3dsx build?

**Comment by Ferk** [comment]
This is probably related to #5337 

@Clydefrosch22 is it still happening after having set properly cores.path ?

**Comment by Clydefrosch22** [comment]
yes, once the core path stuck to the config, favorites began working.

only thing still confusing me is why it could find the cores without the core path set using the normal Load Content > sdmc:/ way of loading. was the core path thing added later?

**Comment by Ferk** [comment]
I don't understand the question. You are saying it was actually possible to find the cores? For me it wasn't.

**Comment by Clydefrosch22** [comment]
what im saying is, when i navigated to a rom through the normal load content menu, at the end it would show me the right core to select and start the game, even with the path not being set.

however, favorites didn't display any cores
while recently played didn't display any games to chose.´



i'm not sure what happens upon clicking load core, i never used that entry.

**Comment by Ferk** [comment]
I bet that this only worked when you loaded a CIA of the corresponding core, which had it already preloaded,
The reason it couldnt be added to any playlist is because it couldn't locate the core that was loaded.

**Comment by gingerbeardman** [comment]
This works for me.

I'm wondering if the @Clydefrosch22 didn't install the required CIAs and/or copied the /retroarch folder to the root of his SD card?
